content/en/continuous_integration/tests/dotnet.md

```diff
@@ -176,6 +176,14 @@ if (scope != null) {
 
 To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][7] section of the .NET custom instrumentation documentation.
 
+### Reporting code coverage
+
+When code coverage is available, the Datadog Tracer (v2.31.0+) reports it under the `test.code_coverage.lines_pct` tag for your test sessions.
+
+If you are using [Coverlet][14] to compute your code coverage, indicate the path to the report file in the `DD_CIVISIBILITY_EXTERNAL_CODE_COVERAGE_PATH` environment variable when running `dd-trace`. The report file must be in the OpenCover or Cobertura formats. Alternatively, you can enable the Datadog Tracer's built-in code coverage calculation with the env var `DD_CIVISIBILITY_CODE_COVERAGE_ENABLED=true`. **Note:** When using Intelligent Test Runner, the tracer's built-in code coverage is enabled by default.
```

Comment:
```suggestion
If you are using [Coverlet][14] to compute your code coverage, indicate the path to the report file in the `DD_CIVISIBILITY_EXTERNAL_CODE_COVERAGE_PATH` environment variable when running `dd-trace`. The report file must be in the OpenCover or Cobertura formats. Alternatively, you can enable the Datadog Tracer's built-in code coverage calculation with the env var `DD_CIVISIBILITY_CODE_COVERAGE_ENABLED=true`. 

**Note**: When using Intelligent Test Runner, the tracer's built-in code coverage is enabled by default.
```
You can also add an alert-info div if this is information you want customers to see

```
  <div class="alert alert-info">When using Intelligent Test Runner, the tracer's built-in code coverage is enabled by default..</div>
```